database/migrations/2014_10_12_100000_create_password_resets_table.php

```diff
@@ -1,31 +0,0 @@
-<?php
-
-use Illuminate\Database\Schema\Blueprint;
-use Illuminate\Database\Migrations\Migration;
-
-class CreatePasswordResetsTable extends Migration
```

Comment:
This has already run on the server, so you're going to need to add a "delete password resets table" migration, not just delete this.